README.md

```diff
@@ -23,3 +23,53 @@ run docker with azure creds:
 
 Output of last command will give you a URL like:
 http://127.0.0.1:8888/lab?token=<--token--> but our notebook will be running on port 10000 as we have mapped the 8888 port of docker container to 10000 port of host machine so browse to http://127.0.0.1:10000/lab?token=<--token--> in browser and **here we go!!**
+
```

Comment:
don't change this readme rather create a new readme inside aws_localstack folder